src/PhpSessionPersistence.php

```diff
@@ -243,12 +244,21 @@ private function createSessionCookie(string $name, string $id, int $cookieLifeti
             FILTER_NULL_ON_FAILURE
         );
 
+        $sameSite = filter_var(
+            ini_get('session.cookie_samesite'),
+            FILTER_VALIDATE_REGEXP,
+            ['options' => ['regexp' => '/(Strict|Lax)/i', 'default' => null]]
```

Comment:
I suggest to assert start and end of string with `^` and `$` and remove unnecessary parens:

```suggestion
            ['options' => ['regexp' => '/^Strict|Lax$/i', 'default' => null]]
```

Reply:
Thanks I removed the regexp as the double validation is indeed not needed